backend/main.py

```diff
@@ -6,8 +6,8 @@
 
 @app.route("/")
 def practice():
-    action = request.args.get("action", "play")
-    return f"Let's {escape(action)}!"
+    name = request.args.get("name", "go")
+    return f"Let's {escape(name)}!"
```

Comment:
These are just place holders anyway, but I'm not following what the intent is here. Could you explain it?

Reply:
there wasnt any. i was just talking myself through them being place holders and putting it back to what i had previously. i had changed it to action what i wasn't understand the .get method